config/engines.yml

```diff
@@ -141,6 +141,14 @@ phpmd:
     - "**.php"
     - "**.module"
     - "**.inc"
+radon:
+  image: codeclimate/codeclimate-radon
+  description: Python tool used to compute Cyclomatic Complexity.
+  community: true
+  enable_regexps:
```

Comment:
@wfleming Ah sorry saw this too late!

I'm happy with either way.

I could open a new PR making that change for all of them if we wanted. WDYT?